README.md

```diff
@@ -18,10 +18,25 @@ $ zig-cache/bin/vulkan-zig-generator path/to/vk.xml output/path/to/vk.zig
 ```
 This reads the xml file, parses its contents, renders the Vulkan bindings, and formats file, before writing the result to the output path. While the intended usage of vulkan-zig is through direct generation from build.zig (see below), the CLI-interface can be used for one-off generation and vendoring the result.
 
+## Using in your project
```

Comment:
This title is now of same importance as the 'Features' part, even though everything below features is also part of that.